docs/docs/references/configuration/cli/trivy_convert.md

```diff
@@ -0,0 +1,52 @@
+## trivy convert
+
+Convert Trivy JSON report into a different format
+
+```
+trivy convert [flags] RESULT_JSON
+```
+
+### Examples
+
+```
+  # report conversion
+  $ trivy image --format json --output result.json --list-all-pkgs debian:11
+  $ trivy convert --format cyclonedx --output result.cdx result.json
```

Comment:
will it work for output produced from trivy `k8s sub-command` ? maybe it should not
when tested it I was getting empty bom:
`trivy k8s cluster --format json --output result.json`
`trivy convert --format cyclonedx --output result.cdx result.json`

```json
{
  "bomFormat": "CycloneDX",
  "specVersion": "1.4",
  "serialNumber": "urn:uuid:c0f5a397-0145-4585-9931-373b4b0e7b3f",
  "version": 1,
  "metadata": {
    "timestamp": "2023-05-24T05:46:02+00:00",
    "tools": [
      {
        "vendor": "aquasecurity",
        "name": "trivy",
        "version": "dev"
      }
    ],
    "component": {
      "type": "",
      "name": "",
      "properties": [
        {
          "name": "aquasecurity:trivy:SchemaVersion",
          "value": "0"
        }
      ]
    }
  },
  "components": [],
  "dependencies": [
    {
      "ref": "",
      "dependsOn": []
    }
  ],
  "vulnerabilities": []
}
```

Reply:
Yeah, nice catch. I forgot to add notes that k8s and aws results don't work.